tests/unit/test_options_config.py

```diff
@@ -218,3 +219,25 @@ def test_parse_config_ignores_unknowns(tmp_path, opt_manager, caplog):
 def test_load_config_missing_file_raises_exception(capsys):
     with pytest.raises(exceptions.ExecutionError):
         config.load_config("foo.cfg", [])
+
+
+@pytest.mark.parametrize("option", ("ignore", "extend-ignore"))
+def test_invalid_ignore_codes_raise_error(tmpdir, opt_manager, option):
+    invalid_option = "//comment"
```

Comment:
DRY here is bad -- it's much harder to understand what's going on in the test -- https://testing.googleblog.com/2014/07/testing-on-toilet-dont-put-logic-in.html

Reply:
Cool, never thought about the fact that KISS is more important than DRY for testing. Learning a lot here 😄 really appreciate the comments.

I removed the `invalid_option` variable but did keep the others as they're imported from elsewhere or originate from the `mark.parametrize` decorator.